fft/src/KokkosFFT_OpenMP_plans.hpp

```diff
@@ -121,7 +121,9 @@ template <typename ExecutionSpace, typename InfoType,
           std::enable_if_t<
               std::is_same_v<ExecutionSpace, Kokkos::DefaultHostExecutionSpace>,
               std::nullptr_t> = nullptr>
-void _destroy_info(InfoType& plan) {}
+void _destroy_info(InfoType&) {
```

Comment:
This info is only needed for `rocfft`. May be better to define a `_destroy_plan_and_info` method to remove no-op function. I will do that in a future cleanup